usaspending_api/references/v2/views/award_types.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from usaspending_api.awards.v2.lookups.lookups import (
+    contract_type_mapping,
+    direct_payment_type_mapping,
+    grant_type_mapping,
+    idv_type_mapping,
+    loan_type_mapping,
+    other_type_mapping,
+)
+
+
+class AwardTypeGroups(APIView):
+    """
+    This route returns a JSON object describing the award type groupings.
+    """
+
+    endpoint_doc = "usaspending_api/api_contracts/contracts/v2/references/award_types.md"
+
+    def get(self, request, format=None):
+        return Response(
+            {
+                "results": {
```

Comment:
I agree with Tony's old comment: This isn't a search, so I don't think it's productive to put it in a results array. The ticket mentions an `award_type_groups` key serving this purpose.

Reply:
Is it necessary to nest the award categories under a key? Each category could be a top-level key